packages/communication-react/CHANGELOG.md

```diff
@@ -11,7 +11,51 @@ Mon, 16 Aug 2021 21:18:19 GMT
 
 ### Changes
 
-- Bump @internal/calling-component-bindings to v1.0.0-beta.4 ([PR #645](https://github.com/azure/communication-ui-library/pull/645) by alcail@microsoft.com)
+* Add remove participant button to the Meeting Composite and Call Composite
+
+* Add applicationID to chat user agent
+
+* Added rtl property to FluentThemeProvider
+
+* Remove redundant prop VideoTile.isVideoReady
+
+* Bump Typescript version to 4.3.5
+
+* Update Call composite adapter joinCall return type to be the same as startCall
+
+* Add explicit string literals for error targets in Chat Stateful client
+
+* Update createAzureCommunicationChatAdapter and createAzureCommunicationCallAdapter constructors to take in a named object instead of seperate args
+
+* Add meeting composite
+
+* Add system message for selector props
+
+* Replace StatefulChatClient.clearErrors() with modifier pattern
+
+* Exposed locale prop to localize composites
```

Comment:
Can this be called out more and moved to the top of this list (we haven't mentioned it in the previous changelog). Maybe: 
```suggestion
* Add Localization to Composites. Learn more: https://azure.github.io/communication-ui-library/?path=/story/localization--page
```

(Can leave out the learn more if it seems strange to include links that could potentially be broken later on)